### DISCUSSION TITLE
splitChannels option in v7Beta?

### DISCUSSION THREAD

**Discussion by Elgoritm** [katspaugh/wavesurfer.js#2790] [Ideas]
Hey there!

Thank you for all the hard work that has been put into the dev and maintenance of wavesurfer-js. I was wondering if splitChannels will come back in v7Beta ? 

ciao!

**Comment by katspaugh** [comment]
Yes!
Added an issue for it: https://github.com/katspaugh/wavesurfer.js/issues/2794
Live example: https://wavesurfer-js.org/examples/#split-channels.js

**Reply by Elgoritm** [reply]
Thank you!
